packages/argo-run/src/dev.ts

```diff
@@ -220,7 +220,7 @@ export async function dev(...args: string[]) {
     // we do not want). This was the only approach I found that works —
     // pass `logLevel`, which is not actually typed as being available, but
     // is used when creating the logger :/
-    ...({logLevel: 'silent'} as any),
+    ...({logLevel: 'error'} as any),
     clientLogLevel: 'silent',
```

Comment:
I don't see a use case of this error. We disable inline mode so that webpackDevServer doesn't inject anything extra for reloading. This is according to the docs

```
        /**
         * When using inline mode, the console in your DevTools will show you
         * messages e.g. before reloading, before an error or when Hot Module
         * Replacement is enabled. This may be too verbose.
         *
         * 'none' and 'warning' are going to be deprecated at the next major
         * version.
         */
         clientLogLevel?: 'silent' | 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'none' | 'warning';
```

Script failed to compile already has enough error output like below.
```
🔭 > Build failed with errors:
ERROR in ./index.ts
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /Users/henrytao/src/github.com/Shopify/tmp/test-checkout-web/index.ts: Unexpected token, expected "," (26:1)

  24 |   root.appendChild(formLayout);
  25 |   root.mount();
> 26 | };
     |  ^
  27 |
```

Reply:
Ah, right, those are errors we don't care about. I see.

Reply:
Thanks for pasting extra info from the docs